### PR TITLE
Documentation Improvements 📖

### PR DESCRIPTION
**Pull request by rikatz**
Signed-off-by: Ricardo Pchevuzinske Katz <ricardo.katz@gmail.com>

**What this PR does / why we need it**: This is a minor documentation improvement, replacing the unnecessary 'tr -d' from documentation to base64 -w0 to not wrap secrets

### CODE REVIEW

File: docs/book/src/clusterctl/overview.md [hunk 1]
```diff
@@ -100,10 +100,10 @@ See the [AWS Provider Prerequisites](https://github.com/kubernetes-sigs/cluster-
 
 ```bash
 # Create the base64 encoded credentials
-export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
-export AZURE_TENANT_ID_B64="$(echo -n "$AZURE_TENANT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
+export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 -w0)"
```
Comment (vincepri) [inline docs/book/src/clusterctl/overview.md]: Unfortunately this doesn't work on Mac, and this is the best way we found to work across operating systems without installing extra dependencies.

Comment (rikatz) [reply docs/book/src/clusterctl/overview.md]: Ah ok, closing then :)

Comment (detiber) [inline docs/book/src/clusterctl/overview.md]: Is `-w0` available on OS X systems? If not, we may want to keep the previous use of `tr`.

Comment (rikatz) [reply docs/book/src/clusterctl/overview.md]: It seems it isn't, didn't know there was this difference between Linux and Mac so already closed it :)

Thank you!

**Pull request closed without merging**